build.sbt

```diff
@@ -1,17 +1,17 @@
-name := "dfdl-mil-std-2045"
- 
-organization := "com.tresys"
- 
-version := "0.0.7"
- 
-scalaVersion := "2.12.11"
- 
-libraryDependencies ++= Seq(
-  "org.apache.daffodil" %% "daffodil-tdml-processor" % "2.6.0" % "test",
-  "com.novocode" % "junit-interface" % "0.11" % "test",
-  "junit" % "junit" % "4.12" % "test",
-)
+lazy val root = (project in file(".")).
+  settings(
+    inThisBuild(List(
+      organization := "com.owlcyberdefense",
+      version      := "1.0.0",
+      scalaVersion := "2.12.15",
+      crossPaths := false,
+      testOptions += Tests.Argument(TestFrameworks.JUnit, "-v"),
+    )),
+    name := "dfdl-mil-std-2045",
+    libraryDependencies := Seq(
+      "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.3.0-SNAPSHOT" % "test",
+      "junit" % "junit" % "4.13.2" % "test",
+      "com.github.sbt" % "junit-interface" % "0.13.2" % "test"
+    )
+  )
```

Comment:
We don't want to use this format. Even though it's the new preferred style according to sbt, it makes it difficult to drop in this schema as a subproject to an existing sbt project, like we do in our regression suite.